backend/pkg/modules/internal/analytics_core/init.go

```diff
@@ -151,6 +129,18 @@ func (p *AnalyticsCore) RegisterAnalyticsModuleHandler(topic TopicType, proccFun
 
 	p.proccFuncRegistered[topic] = append(p.proccFuncRegistered[topic], proccFunc)
 	p.proccFuncRegistered[topic] = orderHandlerFuncsByPriority(p.proccFuncRegistered[topic])
+
+	// Adds dataframes for this new registered function
+	p.dataFramesRegistered[proccFunc] = &ProcFuncDataFrames{dataFrames: map[int]dataframe.DataFrame{}}
```

Comment:
The data frames shall not be automatically generated, but rather generated upon registration of data frame for ProcessingFunction